bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/handler/IPBridgeHandler.java

```diff
@@ -339,9 +341,28 @@ private void parseUpdates() {
                 this.keepAliveReconnect.cancel(true);
             }
 
-            Matcher matcher = STATUS_REGEX.matcher(line);
+            Matcher matcher = RESPONSE_REGEX.matcher(line);
+            boolean responseMatched = matcher.find();
+
+            if (!responseMatched) {
+                // In some cases with Caseta a CLI prompt may be embedded within a received response line.
+                if (line.contains("NET>")) {
+                    // Try to remove it and re-attempt the regex match.
+                    scrubbedLine = line.replaceAll("[GQ]NET> ", "");
+                    matcher = RESPONSE_REGEX.matcher(scrubbedLine);
+                    responseMatched = matcher.find();
+                    if (responseMatched) {
+                        line = scrubbedLine;
+                        logger.debug("Cleaned response line: {}", scrubbedLine);
```

Comment:
Isn't this more a TRACE logging?

Reply:
Hi. Thanks for reviewing this so quickly. I wanted the “cleaned” response lines to be logged at the same level as the normal response lines, which is debug.  It could certainly be argued that all responses should be logged only at trace level, but since they’ve been logged at debug level all along and users are used to it, I didn’t really want to mess with it right now.